### DISCUSSION TITLE
Tailwind UI and Tailwind CSS V3 ?

### DISCUSSION THREAD

**Discussion by bahdcoder** [tailwindlabs/tailwindcss#7648] [Help]
Hello. Just went through the guide for Tailwind UI, and there's no mention of the package `@tailwindcss/ui`. I later noticed this package hasn't been updated in over a year and still uses outdated versions of @tailwindcss/typography. 

I went ahead to proceed using Tailwind UI without this package, but noticed none of the colours are available and the copied components do not match in my app.

Am I please missing something? Please help me understand how to set up Tailwind UI and Tailwind css v3. Thanks thanks.

**Comment by simonswiss** [comment]
Hey!

It's been a fair while since Tailwind UI doesn't rely on an additional package (`@tailwindcss/ui`) – all it needs to work is Tailwind CSS.

Some components will require specific plugins (Forms, Typography, Aspect Ratio, ...), and this will be flagged in the code comment at the top of the component.

The colors should work out of the box with your project - unless you're using some customized colors in your Tailwind config ported over from Tailwind CSS version 2?

Are you able to share at least your Tailwind config file, and ideally a little code repo with all your setup (not the whole app with all the pages, just your barebones setup?

Happy to help!

**Reply by bahdcoder** [reply]
Heey thanks a lot for your reply. My config is a really basic one. Here goes:

```js
const defaultTheme = require('tailwindcss/defaultTheme')

module.exports = {
  content: ['./resources/js/**/*.tsx', './resources/views/app.edge'],
  theme: {
    extend: {
      fontFamily: {
        body: ['Space Grotesk', ...defaultTheme.fontFamily.serif],
      },
    },
  },
  plugins: [
    // require('@tailwindcss/ui'),
    require('@tailwindcss/forms'),
    require('@tailwindcss/typography'),
    require('@tailwindcss/aspect-ratio'),
  ],
}

```

Change nothing relating to colours. Also disabled `@tailwindcss/ui` when I realised its outdated and possibly not needed anymore. 

I checked to make sure all the files were discoverable, but most colours won't just show up. 

Happy to hear any thoughts you might have. Thanks a lot.
